include/psa/crypto_builtin_hash.h

```diff
@@ -0,0 +1,89 @@
+/*
+ *  Context structure declaration of the software-based driver which performs
+ *  hashing through the PSA Crypto driver dispatch layer.
+ */
```

Comment:
Please state that this header and its content are not part of the Mbed TLS API and applications must not depend on it.